modules/registrars/openprovider/Controllers/System/ConfigController.php

```diff
@@ -179,8 +179,8 @@ protected function generateLoginError(array $configarray, $error)
     protected function checkCredentials($configarray, $params)
     {
         $differentHost = $params['test_mode'] == 'on' ?
-            Configuration::get('api_url') :
-            Configuration::get('api_url_cte');
+            Configuration::get('api_url_cte') :
```

Comment:
strange, this worked for long time
please make sure

Reply:
Confirmed! `Configuration::get('api_url_cte')` is for Sandbox/Customer Test Environment